<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="client">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>321</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rule_book">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>551</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="user_desk">
    <property name="geometry">
     <rect>
      <x>-180</x>
      <y>330</y>
      <width>1301</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/tempDesk2A8.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="question_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="evaluation_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NEW CLIENT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ruleBook_pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>270</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:/resources/img/tempBooksA8.png);
background: none;
background-repeat: none;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeRules_pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>client</zorder>
   <zorder>user_desk</zorder>
   <zorder>rule_book</zorder>
   <zorder>question_pushButton</zorder>
   <zorder>evaluation_pushButton</zorder>
   <zorder>pushButton</zorder>
   <zorder>ruleBook_pushButton</zorder>
   <zorder>closeRules_pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
